.github/ISSUE_TEMPLATE/data-problem.yml

```diff
@@ -0,0 +1,80 @@
+name: "Data problem"
+title: "<NAME THE FEATURE> - <SUMMARIZE THE PROBLEM>"
+description: Report incorrect, incomplete, or missing data
+labels: ["needs triage 🔎"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Before you start
+
+        **Are you looking for development help?**
+        ↩ Please note that we cannot provide assistance on web development. We recommend asking around on a dedicated help forum like StackOverflow.
+
+        **Is your issue about content on an MDN page besides the compatibility table?**
+        ↩ Use the _Report a problem with this content on GitHub_ link at the bottom of the page.
+
+        **Need help with a browser?**
+        🙋 To get help with [Firefox](https://support.mozilla.org/en-US/kb/file-bug-report-or-feature-request-mozilla), [Chrome](https://support.google.com/chrome/answer/95315?hl=en-GB&ref_topic=7439544), [Safari](https://www.apple.com/feedback/safari.html), or another browser, check the browser's support site.
+
+        ---
+  - type: dropdown
+    id: type
+    attributes:
+      label: What type of issue is this?
+      options:
+        - label: Incorrect version number (ex. Chrome says "42" but support was added in "40")
```

Comment:
Maybe update the versions here to be more contemporary, and also a bigger error than 2 versions for more realism?